Microsoft.Toolkit.Uwp.UI.Controls/AdaptiveGridView/AdaptiveGridView.Properties.cs

```diff
@@ -129,6 +154,36 @@ public ICommand ItemClickCommand
         }
 
         /// <summary>
+        /// Gets or sets the selected multiple objects in the collection.
+        /// </summary>
+        /// <value>The object that is used to store selected multiple items. Default is an empty IList collection of object.</value>
+        public IList<object> SelectedItems
+        {
+            get { return (IList<object>)GetValue(SelectedItemsProperty); }
+            set { SetValue(SelectedItemsProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether gets or sets whether the items are clickable or not.
+        /// </summary>
+        /// <value>Default is false.</value>
+        public bool IsItemClickEnabled
+        {
+            get { return (bool)GetValue(IsItemClickEnabledProperty); }
+            set { SetValue(IsItemClickEnabledProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the SelectionMode property of ListViewBase. Does not support runtime changes at the moment.
+        /// </summary>
+        /// <value>Default is single which allows users to only pick single item in the collection.</value>
+        public ListViewSelectionMode SelectionMode
```

Comment:
You're right. I thought maybe I'll catch the SelectionModeChanged and override it in my way in the future. I don't know the default behavior on changing selection mode to multiple from single on base but it should keep the first selected item and then change the mode imo. We can discuss about this overriding default behavior stuff. Is there any policy for not changing the behaviors?

I will check the default behavior on that stuff and let you know. Maybe it's already that way and I don't know.